### PR TITLE
Google workspaces admin activity report

### PR DESCRIPTION
**Pull request by jcortes**

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

**Review by feyzullah — DISMISSED** [review]
LGTM

File: components/google_workspace/actions/list-activities-by-admin/list-activities-by-admin.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import "gaxios";
```
Comment (feyzullah) [inline components/google_workspace/actions/list-activities-by-admin/list-activities-by-admin.ts:1]: Is this one used somewhere? It is not in `package.json`, if it is a dependency used can we add it to `package.json` too?

**Review by feyzullah — COMMENTED** [review]
LGTM. I added 1 comment that I am not sure about. Can you please check?

File: components/google_workspace/actions/list-activities-by-admin/list-activities-by-admin.ts (hunk 1 above)
Comment (jcortes) [reply components/google_workspace/actions/list-activities-by-admin/list-activities-by-admin.ts:1]: I'll remove it. Thanks

**Pull request merged**